source/docs/software/basic-programming/java-units.rst

```diff
@@ -68,13 +117,28 @@ Unit conversions can be done by calling ``Measure#in(Unit)``. The Java type syst
      maxAcceleration.in(MetersPerSecondPerSecond)
    );
 
-.. note:: Due to restrictions of the Java type system, acceleration is modeled as ``Velocity<Velocity<X>>>`` instead of having its own class.
+Usage Example
+~~~~~~~~~~~~~
+
+Pulling all of the concepts together, we can create an example that calculates the end effector position of an arm mechanism:
+
+.. code-block:: java
+
+  Measure<Distance> armLength = Feet.of(3).plus(Inches.of(4.25));
+  Measure<Distance> endEffectorX = armLength.times(Math.cos(getArmAngle().in(Radians)));
+  Measure<Distance> endEffectorY = armLength.times(Math.sin(getArmAngle().in(Radians)));
+
+Human-readable Formatting
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``Measure`` class has both ``toShortString()`` and ``toLongString()`` methods that can be used to get a human-readable representation of the measure. This feature is useful to display a measurement on a dashboard or in logs.
+
+Mutibility and Object Creation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Memory Usage and the Garbage Collector
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+To reduce the number of object instances you create, and reduce memory usage, a special ``MutableMeasure`` class is available. You may want to consider using mutable objects if you are using the units library repeatedly, such as in the robot's periodic loop.
 
-The roboRIO is a severely memory-constrained runtime environment and the Java garbage collector has to run to keep memory usage to a reasonable level. The garbage collector will have to pause the robot program in order to free unused objects, which manifests as loop overruns and potentially jittery controls. To avoid this issue, allocate as few ``Measure`` objects as possible in areas of code that will run periodically, such as in a Command's ``execute`` or one of TimedRobot's periodic methods.
```

Comment:
I think this was an important section to keep.

Reply:
Thanks for reviewing, but I disagree. There aren't similar warnings for other classes in WPILib, such as `Translation2d`. At best, this seems like a general comment that belongs someplace else.
See my discussion in topic https://github.com/wpilibsuite/2024Beta/discussions/29

Reply:
Then it should be moved to a more general location instead of deleted outright in this PR.

Reply:
Again, I disagree because it never existed before and just slipped in with the java-units documentation. However, I'll see if I can find a good spot for the warning.

Reply:
Moving this to a general location would be good, but even so I think it would be good to leave in a sentence motivating the use of the mutable methods. Otherwise it could be confusing: "Why is there the option to mutate in the first place? Should I be mutating?"

Reply:
I'm working on adding a page, and I will link it from this documentation. But, I did include such a statement at the beginning of the section: "To reduce the number of object instances you create, and reduce memory usage..."